src/remapper.rs

```diff
@@ -417,13 +480,21 @@ fn is_modifier(key: &KeyCode) -> bool {
 /// Unfortunately the underlying type doesn't allow direct
 /// comparison, but that's ok for our purposes.
 fn modifiers_first(a: &KeyCode, b: &KeyCode) -> Ordering {
-    if is_modifier(a) {
-        if is_modifier(b) {
+    let mut a_ismod = false;
+    let mut b_ismod = false;
+    if let EventCode::EV_KEY(k) = a {
+       a_ismod = is_modifier(&k);
+    }
+    if let EventCode::EV_KEY(k) = b {
+       b_ismod = is_modifier(&k);
+    }
+    if a_ismod && b_ismod {
+        if b_ismod {
```

Comment:
this feels harder to read than it was before. Please revert `is_modifier` back to its former state (accepting `&KeyCode`) to resolve this.